Update Featured Artists and add Mappers' Guild (#3280)
ppy/osu-wiki: wiki/Mappers_Guild/en.md

### Automation

While the initial plan for the Mappers' Guild had always been to have MMO elements like quests, that was found to be impractical to handle while manually managing deadlines for the limited number of guild participants.

Development of a website to automate Mappers' Guild began at the end of 2018, allowing for more adequate distribution of rewards and management of quest/party systems.

The [Mappers' Guild website](https://mappersguild.com/) launched on March 1st, 2019, and is where everything about the Mappers' Guild is currently controlled. Since being opened to the public, the guild has reached a member count over 500 and publishes news posts such as [this](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) to keep the rest of the osu! community up to date with featured artist mapping content.

## System

Mappers with 3 or more ranked maps can join the guild at any time via the [Mappers' Guild website](https://mappersguild.com/) by authenticating their osu! account. Anyone else is free to view the website's quest and mapping activities.

> Mappers can earn rewards by participating in the ranking process for featured artist maps that are connected to the Mappers' Guild. This primarily focuses on creating beatmaps, however rewards are also provided for users who contribute by modding, creating storyboards, and complete quests within the guild.

Reviewer: ```suggestion
Mappers can earn rewards by participating in the ranking process for featured artist maps that are connected to the Mappers' Guild. This primarily focuses on creating beatmaps, however rewards are also provided for users who contribute by modding, creating storyboards, and completing quests within the guild.
```

### Parties

Parties are groups of mappers who register to complete quests together. Mappers may join as many parties as they want at a time.

Special quests require higher guild ranks to participate in. For a party to qualify as a higher rank, at least half of its members must be the required rank or higher.

> ### Quests

Reviewer: you could add a sentence or two describing the new features of how quests cost points to enter and that they may expire after a period of time

Quests reward mappers for following certain requirements when creating their featured artist maps. Usually quests involve creating a certain number of beatmaps for a new featured artist, however some quests involve small gimmicks.

Each quest has its own party size specifications and deadline. Failing to meet a deadline or members leaving the party mid-way through can result in the quest being dropped, lowering a user's points earnings or even penalizing them.

Quest participation is not required for members of the Mappers' Guild. If a user wants to, they may create beatmaps alone to earn rewards.

### Rewards

Rewards are distributed through a points system. When a user reaches a specified points threshold, they earn a different rank signified by a profile badge. There are three profile badge tiers at 100, 250, and 500 points respectively.

> Points are earned for different mapping-related tasks. Earnings vary between tasks, though they are generally weighted according to the amount of time required. For example, mapping a `Hard` difficulty takes less time than mapping an `Easy` difficulty, so the first rewards more points than the second. 

Reviewer 2: this should actually be the opposite tho...

"For example, mapping a `Hard` difficulty takes more than mapping an `Easy` difficulty, so the first rewards more points than the second."

> Difficulty creation tasks earn a user more points if the beatmap is associated with a quest. Completing a quest provides users with a pre-determined bonus that varies between each quest.

Reviewer: ```suggestion
Difficulty creation tasks earn a user more points if the beatmap is associated with a quest. Completing a quest on time provides users with a pre-determined bonus.
```

since you said it's a flat value now
